src/mbgl/layout/symbol_layout.cpp

```diff
@@ -695,8 +695,11 @@ void SymbolLayout::addFeature(const std::size_t layoutFeatureIndex,
         }
     } else if (type == FeatureType::LineString) {
         for (const auto& line : feature.geometry) {
-            Anchor anchor(line[0].x, line[0].y, 0, minScale);
-            addSymbolInstance(anchor, createSymbolInstanceSharedData(line));
+            // Skip invalid LineStrings.
```

Comment:
nit:  `if (line.empty()) continue;`

Reply:
Ah, that's better. My C++ is out of practice. 😄

Reply:
Pushed.